tests/cpp/cell/test_copy.cu

```diff
@@ -1,9 +1,95 @@
-#include "tiled_cuda_test.h"
+#include "cell/copy/dyn_copy.hpp"
+#include "cell/sync.hpp"
+#include "cell/traits/copy.hpp"
+#include "common/test_utils.hpp"
+
+#include <thrust/device_vector.h>
+#include <thrust/host_vector.h>
+
+namespace traits = tiledcuda::cell::traits;
 
 namespace tiledcuda {
+
+namespace {
```

Comment:
It seems unnecessary to have the `namespace`?

Reply:
My intention is to place the code that should not be referenced outside of this file into an anonymous namespace. But I think the current organization is still experimental, I remove the anonymous namespace.